src/coreclr/src/vm/eventpipe.cpp

```diff
@@ -67,6 +70,8 @@ void EventPipe::Initialize()
 
     s_config.Initialize();
 
+    s_ResumeRuntimeStartupEvent->CreateManualEvent(false);
```

Comment:
Can we initialize this event only when we know we will need to be waiting for it?